src/Pages/ForgotPassPage/ForgotPassPage.js

```diff
@@ -0,0 +1,38 @@
+import CardWrapper from '../../Atoms/CardWrapper/CardWrapper';
+import InputField from '../../Atoms/InputField/InputField';
+import PriSecBtn from '../../Atoms/PriSecBtn/PriSecBtn';
+import { useState } from 'react';
+import StyledForgotPassForm from './ForgotPassPageStyled';
+import { useNavigate } from 'react-router-dom';
+
+const ForgotPassPage = () => {
+  const [email, setEmail] = useState('');
+  const navigate = useNavigate();
+
+  const submitHandler = (event) => {
```

Comment:
maybe it will be good to add some confirmation for user that mail was sent?

Reply:
in future im sure, but to check and confirm i think we need DB. Maybe another page UI ? Confirmation Page. It's doable.

Reply:
added as TODO to Sprint I